src/main/java/bio/terra/cli/command/resource/create/GcpDataprocCluster.java

```diff
@@ -0,0 +1,454 @@
+package bio.terra.cli.command.resource.create;
+
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.command.shared.WsmBaseCommand;
+import bio.terra.cli.command.shared.options.Format;
+import bio.terra.cli.command.shared.options.ResourceCreation;
+import bio.terra.cli.command.shared.options.WorkspaceOverride;
+import bio.terra.cli.serialization.userfacing.input.CreateGcpDataprocClusterParams;
+import bio.terra.cli.serialization.userfacing.input.CreateGcpDataprocClusterParams.NodeConfig;
+import bio.terra.cli.serialization.userfacing.input.CreateResourceParams;
+import bio.terra.cli.serialization.userfacing.resource.UFGcpDataprocCluster;
+import bio.terra.cli.utils.CommandUtils;
+import bio.terra.workspace.model.AccessScope;
+import bio.terra.workspace.model.CloningInstructionsEnum;
+import bio.terra.workspace.model.CloudPlatform;
+import bio.terra.workspace.model.GcpDataprocClusterCreationParameters.SoftwareFrameworkEnum;
+import bio.terra.workspace.model.GcpDataprocClusterInstanceGroupConfig.PreemptibilityEnum;
+import bio.terra.workspace.model.StewardshipType;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import picocli.CommandLine;
+
+/**
+ * This class corresponds to the fourth-level "terra resource create gcp-dataproc-cluster" command.
+ */
+@CommandLine.Command(
+    name = "gcp-dataproc-cluster",
```

Comment:
It was because vertex ai notebooks were `gcp-notebook`. I can remove the prefix since dataproc kind of already is the prefix. We miight have to support a similar managed spark cluster resource in aws.